prov/tcp/src/tcpx_attr.c

```diff
@@ -57,6 +63,7 @@ static struct fi_tx_attr tcpx_tx_attr = {
 
 static struct fi_rx_attr tcpx_rx_attr = {
 	.caps = TCPX_EP_CAPS | TCPX_RX_CAPS,
+	.op_flags = TCPX_RX_OP_FLAGS,
```

Comment:
@vkrishna can you please verify if these changes look good for tcp?

Reply:
They look good. Thanks